### PR TITLE
Add CI tests

### PR DESCRIPTION
**Pull request by danielsdeboer**

### CODE REVIEW

File: src/OdbcServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        OdbcConnection::resolverFor(
+            'odbc',
+            /**
+             * @param \PDO|\Closure $connection
+             * @param string $database
+             * @param string $prefix
+             * @param array $config
+             * @return \Dbt\Odbc\OdbcConnection
+             */
+            function ($connection, $database, $prefix, $config) {
```
Comment (kaydenvanrijn) [inline src/OdbcServiceProvider.php]:
```suggestion
<?php

namespace Dbt\Odbc;

use Illuminate\Support\ServiceProvider;

class OdbcServiceProvider extends ServiceProvider
{
    public function boot(): void
    {
        OdbcConnection::resolverFor(
            'odbc',
            function (\PDO|\Closure $connection, string $database, string $prefix, array $config): OdbcConnection {
```

File: composer.json [hunk 2]
```diff
@@ -37,20 +37,22 @@
...
         "psr-4": {
-            "Dbt\\Odbc\\Tests\\": "tests"
+            "Dbt\\Odbc\\Tests\\": "tests/"
         }
     },
     "extra": {
         "laravel": {
             "providers": [
-                "Dbt\\Odbc\\Provider"
+                "OdbcDriverServiceProvider"
```
Comment (kaydenvanrijn) [inline composer.json]: This doesn't need the full class name?

**Review by kaydenvanrijn — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (danielsdeboer) [reply composer.json]: It sure does.

File: src/OdbcServiceProvider.php (hunk 1 above)
Comment (danielsdeboer) [reply src/OdbcServiceProvider.php]: That's how it's done in (for instance) the SQLite driver. I don't want to fafo on this.

**Review by kaydenvanrijn — APPROVED** [review]

**Pull request merged**